Makefile

```diff
@@ -60,7 +60,8 @@ build:  ## Build docker image
 		build parrot-app
 
 envfile:  ## Generate env file with variables with prefix PARROT_
-	env | egrep ^PARROT_ > .gen.env
+	$(shell env | egrep ^PARROT_ > .gen.env)
+	$(shell test -f .env && cat .env > .gen.env)
```

Comment:
It looks like `test -f .env` will be always true.

Reply:
No
```
╭─lowit@boojum ~/Downloads/test 
╰─$ ls -a 
.  ..  .env
```

```
╭─lowit@boojum ~/Downloads/test 
╰─$ test -f .env; echo $?
0
╭─lowit@boojum ~/Downloads/test 
╰─$ test -f .no_file; echo $?
1
```

Reply:
Sorry, I missed that previous step creates `.gen.env` and that check is for `.env`, not `.gen.env`.